From 76407e1ebfae6f45e44a8779a842892682e6e27a Mon Sep 17 00:00:00 2001
From: Alxv <avong2011@gmail.com>
Date: Tue, 11 Apr 2023 15:37:36 -0400
Subject: [PATCH] style: updated menu labels

---
 source/menu.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/source/menu.c b/source/menu.c
index ef9c883..af87bba 100644
--- a/source/menu.c
+++ b/source/menu.c
@@ -656,15 +656,15 @@ static int main_menu_handler(int id, int keys)
 
 static menu_entry e_menu_main[] =
 {
-	mee_handler_id("Resume game",        MA_MAIN_RESUME_GAME, main_menu_handler),
-	mee_handler_id("Save state",         MA_MAIN_SAVE_STATE,  main_menu_handler),
-	mee_handler_id("Load state",         MA_MAIN_LOAD_STATE,  main_menu_handler),
-	mee_handler_id("Disc control",       MA_MAIN_DISC_CTRL,   menu_loop_disc),
+	mee_handler_id("Resume",        MA_MAIN_RESUME_GAME, main_menu_handler),
+	mee_handler_id("Save",         MA_MAIN_SAVE_STATE,  main_menu_handler),
+	mee_handler_id("Load",         MA_MAIN_LOAD_STATE,  main_menu_handler),
+	mee_handler_id("Load Disc",       MA_MAIN_DISC_CTRL,   menu_loop_disc),
 	mee_handler_id("Cheats",             MA_MAIN_CHEATS,      menu_loop_cheats),
 	mee_handler   ("Options",            menu_loop_options),
-	mee_handler_id("Reset game",         MA_MAIN_RESET_GAME,  main_menu_handler),
-	mee_handler_id("Load new game",      MA_MAIN_CONTENT_SEL, menu_loop_select_content),
-	mee_handler_id("Exit",               MA_MAIN_EXIT,        main_menu_handler),
+	mee_handler_id("Reset",         MA_MAIN_RESET_GAME,  main_menu_handler),
+	mee_handler_id("New game",      MA_MAIN_CONTENT_SEL, menu_loop_select_content),
+	mee_handler_id("Quit game",               MA_MAIN_EXIT,        main_menu_handler),
 	mee_end,
 };
 
-- 
2.39.2 (Apple Git-143)

